tests/integration/construct_component/testcomponent-go/main.go

```diff
@@ -41,7 +41,7 @@ func NewResource(ctx *pulumi.Context, name string, echo pulumi.Input,
 	opts ...pulumi.ResourceOption) (*Resource, error) {
 	args := &ResourceArgs{Echo: echo}
 	var resource Resource
-	err := ctx.RegisterResource("testcomponent:index:Resource", name, args, &resource, opts...)
+	err := ctx.RegisterResource(providerName+":index:Resource", name, args, &resource, opts...)
```

Comment:
I don't see providerName declared in here?

Reply:
Its declared on line 104. 
https://github.com/pulumi/pulumi/blob/12c75e91dbe3b9b8ea0ce0e043d6fbfdc9a19a06/tests/integration/construct_component/testcomponent-go/main.go#L104